docs/apm/correlations.asciidoc

```diff
@@ -21,53 +19,60 @@ Or, perhaps set of users, based on IP address or region, is facing increased lat
 With a service selected, click **View correlations**:
 
 [role="screenshot"]
-image::apm/images/correlations.png[Correlations]
+image::apm/images/correlations.png[View correlations]
+//TBD: Is this section and screenshot still useful for orienting the user?
 
-Queries within the APM app apply to the correlations shown in the correlations fly-out.
-
-If a correlated field seems noteworthy, use the **Filter** quick links:
-
-* `+` creates a new query in the APM app for filtering transactions containing the selected value.
-* `-` creates a new query in the APM app to filter out transactions containing the selected value.
+NOTE: Queries within the APM app are also applied to the correlations.
 
 [discrete]
 [[correlations-latency]]
 ==== Find high-latency correlations
 
-Correlations help you discover which fields are contributing to increased service latency.
-
-A latency distribution chart visualizes the overall latency of the selected service's transactions.
-Correlated attributes are sorted by _Impact_–a visual representation of the
-{ref}/search-aggregations-bucket-significantterms-aggregation.html[significant terms aggregation]
-score that powers correlations.
-Attributes with a high impact, or attributes present in a large percentage of slow transactions,
-may contribute to increased latency.
-
-To find high-latency correlations, hover over each potentially correlated attribute to
-compare the latency distribution of transactions with and without the selected attribute.
-
-For example, in the screenshot below, the field `user_agent.name` and value `HeadlessChrome`
-exists primarily in higher-latency transactions between 3.7 and 8.7 seconds.
+The correlations on the *Latency* tab help you discover which attributes are
+contributing to increased service latency.
 
 [role="screenshot"]
-image::apm/images/correlations-hover.png[Correlations hover effect]
-
-Select the `+` filter to create a new query in the APM app for transactions with
-`user_agent.name: HeadlessChrome`. With the "noise" now filtered out,
+image::apm/images/correlations-hover.png[Latency correlations]
+
+A latency distribution chart visualizes the overall latency of the service. If
+there are attributes that have a statistically significant impact on the
+latency, they are listed in a table below the chart. The table is sorted by
```

Comment:
~~latency~~
high latency

Reply:
I've changed the phrasing of this paragraph to "If there are attributes that have a statistically
significant correlation with slow response times..." since I have to keep reminding myself what latency means. If you prefer "impact on high latency", just let me know.